app/Http/Controllers/Settings/BankDataController.php

```
@@ -0,0 +1,372 @@
+    private GoCardlessBankData $client;
+
+    private User $user;
+
+    /**
+     * Initializes the controller with the authenticated user's GoCardless credentials and sets up the GoCardless API client.
+     *
+     * Retrieves the current user's GoCardless credentials from the database. If credentials are present, initializes the GoCardlessBankData client; otherwise, logs a warning.
+     */
+    public function __construct()
+    {
+        $this->user = User::select(
```

PHPStan (2.1.15)

Call to an undefined static method App\Models\User::select().


(staticMethod.notFound)